src/patternfly/components/LabelGroup/label-group.scss

```diff
@@ -110,31 +115,31 @@
   flex: 1;
   flex-wrap: wrap;
   align-items: baseline;
+  row-gap: var(--pf-c-label-group__main--RowGap);
+  column-gap: var(--pf-c-label-group__main--ColumnGap);
 }
 
 .pf-c-label-group__list {
   display: inline-flex;
   flex-wrap: wrap;
-  margin-right: var(--pf-c-label-group__list--MarginRight);
-  margin-bottom: var(--pf-c-label-group__list--MarginBottom);
+  row-gap: var(--pf-c-label-group__list--RowGap);
+  column-gap: var(--pf-c-label-group__list--ColumnGap);
 }
 
 .pf-c-label-group__list-item {
   display: inline-flex;
-  margin-right: var(--pf-c-label-group__list-item--MarginRight);
-  margin-bottom: var(--pf-c-label-group__list-item--MarginBottom);
 }
 
 .pf-c-label-group__label {
   @include pf-text-overflow;
 
   max-width: var(--pf-c-label-group__label--MaxWidth);
-  margin-right: var(--pf-c-label-group__label--MarginRight);
-  margin-bottom: var(--pf-c-label-group__label--MarginBottom);
   font-size: var(--pf-c-label-group__label--FontSize);
 }
 
 .pf-c-label-group__close {
+  display: flex;
+  align-self: start;
```

Comment:
I think it's fine, but it's worth noting that this shifts the close button up by a pixel or two?

This PR:
![patternfly-pr-5296 surge sh_components_label-group_html_category-removable_](https://user-images.githubusercontent.com/35148959/214460195-78f26e8a-caae-41fb-8958-ccd4f78c0daa.png)

Currently on v4:
![pf4 patternfly org_components_label-group_html_category-removable_](https://user-images.githubusercontent.com/35148959/214460232-ea0a2c91-0b59-4279-91e3-b7d33bcf8790.png)

And that's from this change to move the close button to the top instead of being vertically centered when labels wrap. I didn't see that called out in the issue, so just wanted to make sure that change is good, too:

This PR:
![pf4 patternfly org_components_label-group_html_category-removable_ (1)](https://user-images.githubusercontent.com/35148959/214460492-d1000938-3bb1-4070-ab79-d49730bf03df.png)

Currently on v4:
![patternfly-pr-5296 surge sh_components_label-group_html_category-removable_ (1)](https://user-images.githubusercontent.com/35148959/214460481-d855b2e2-792b-4255-894d-03c2721f5499.png)

cc @mceledonia @mmenestr

Reply:
I personally think moving it to the top right is the right choice!
I'm confused though because you say you moved the close button to the top, but image is showing that that is the current version?

Reply:
Just to clarify - this PR maintains the close button at the top right that was changed in a previous PR.
<img width="354" alt="image" src="https://user-images.githubusercontent.com/19825616/214873108-c8a5dad9-aa1c-4f6f-a802-a1c9ee637620.png">